gha/bin/release-init.py

```diff
@@ -221,6 +227,36 @@ def handle(self, base: str, head: str) -> NoReturn:
                     inwardIssue=issue_key,
                     outwardIssue=release_key,
                 )
+        # Check for previous cancelled release
+        cancelled_release_scope = []
+        if last_cancelled_release:
```

Comment:
Прошлый релиз всегда есть - отменённый или опубликованный.

```suggestion
        if last_release:
```